website/source/docs/configuration/storage/foundationdb.html.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: "docs"
+page_title: "FoundationDB - Storage Backends - Configuration"
+sidebar_current: "docs-configuration-storage-foundationdb"
+description: |-
+  The FoundationDB storage backend is used to persist Vault's data in the
+  FoundationDB KV store.
+---
+
+# FoundationDB Storage Backend
+
+The FoundationDB storage backend is used to persist Vault's data in
+[FoundationDB][foundationdb] table.
+
+- **High Availability** – the FoundationDB storage backend supports high
+  availability. The HA implementation relies on the clocks of the Vault
+  nodes inside the cluster being properly sychronized; clock skews are
+  susceptible to cause issues.
+
+```hcl
+storage "foundationdb" {
+  api_version  = 510
+  cluster_file = "/path/to/fdb.cluster"
+  path         = "vault-top-level-directory"
```

Comment:
Please add an `ha_enabled` flag and gate HA support on it. See e.g. dynamo for guidance.